### ISSUE TITLE
[Indigo] undefined symbol in planning_scene_monitor

### ISSUE THREAD

**Issue by rbbg** [ros-planning/moveit#456]
I am running into the following error when trying to call `planning_scene_monitor::getStateUpdateFrequency()`:

```
/opt/ros/indigo/lib/moveit_ros_move_group/move_group: symbol lookup error:
  /home/ruben/cohesion/iiwa_ws/devel/lib/libmoveit_rt_optim.so: undefined symbol:
   _ZN22planning_scene_monitor20PlanningSceneMonitor23getStateUpdateFrequencyEv
```

This suggests a linking error, but `planning_scene_monitor::setStateUpdateFrequency(double)` _does_ work, so it seems like the symbol is missing from this version of the library.

My environment:
* ROS Distro: Indigo
* OS Version: Ubuntu 14.04
* Binary build -> Linked against `libmoveit_planning_scene_monitor.so.0.7.3`

**Comment by v4hn** [comment]
does rebuilding your `/home/ruben/cohesion/iiwa_ws/` workspace solve the problem?

**Comment by rbbg** [comment]
Found the problem: the function is only [declared but never defined](https://github.com/ros-planning/moveit/search?utf8=%E2%9C%93&q=getStateUpdateFrequency). Should be easy to fix :). I'll put in a PR.

**Comment by v4hn** [comment]
Wow.

A Pull-Request for this would be great.

**Comment by v4hn** [comment]
resolved in #463
